### PR TITLE
upgrade ntfs to version 0.12.0

### PR DESCRIPTION
**Pull request by patochectp**
ref: ND-1496

### CODE REVIEW

**Review by prhod — APPROVED** [review]

**Comment by patochectp** [comment]
ref: ND-1496

File: ntfs_fr.md [hunk 1]
```diff
@@ -336,6 +337,7 @@ stop_timezone | timezones | Optionnel | Fuseau horaire, se référer à http://e
 equipment_id | chaine | Optionnel | Identifiant de la propriété accessibilité
 level_id | chaine | Optionnel | lien vers un niveau décrit dans le fichier [`levels.txt`](#levelstxt-optionnel)
 platform_code | chaine | Optionnel | Identifiant de la plateforme d'un arrêt (par exemple `G` ou `3`). Ne peut être renseigné que pour les arrêts physiques (`location_type=0`) ou les zones d'embarquements (`location_type=5`)
+address_id | chaine | Optionnel | Identifiant de l'adresse de l'arrêt (lien vers le fichier [`addresses.txt`]). Ce champ ne s'applique que sur les arrêts physiques (location_type = 0)
```
Comment (woshilapin) [inline ntfs_fr.md]:
```suggestion
address_id | chaine | Optionnel | Identifiant de l'adresse de l'arrêt (lien vers le fichier [`addresses.txt`]). Ce champ ne s'applique que sur les arrêts physiques (`location_type=0`)
```

**Review by woshilapin — APPROVED** [review]

**Review by xlqian — APPROVED** [review]

**Comment by SGrenet** [comment]
After discussion, it seems Titan give the house_number as an integer, and navitia interface has the type integer for house_number.
Then giving the root Titan house_number value avoid re-processing it in navitia.

Could you giving house_number as an integer and an other field for complete house_number?
For exemple full_house_number or long_house_number or whatever

**Comment by SGrenet** [comment]
it's not the correct PR but, house_number must be a non zero value

**Comment by prhod** [comment]
Thanks a lot for the review :+1: 
As discussed IRL, NTFS doesn't have a constraint about house_number, it's a prerequisite of the Navitia API that probably needs to be applied on OSM data either. Unless there is a more global need to have a constraint, I'd like to keep it that way :)

**Review by prhod — APPROVED** [review]

**Pull request merged**